pro/admin-tasks.md

```diff
@@ -0,0 +1,696 @@
+---
+layout: pro
+title: Administrator tasks
+order: 4
+user-guide: true
+---
+
+# Administrator tasks
+
+
+If you find that you cannot perform any of the tasks detailed below when logged into your FixMyStreet account, it is likely that your account has not been given the relevant permissions.
+
+Contact your account administrator to request these permissions. Accompanying each task covered in this document, you will see ‘permissions required’, listing the checkboxes an Administrator must tick on your account for you to be able to perform this task.
+
+<div class="pro-task" markdown="1" id="navigating-as-pro-staff">
+
+### Navigating FixMyStreet Pro as a staff user
+
+<img alt="When you're loggedin as an administrator the 'admin' button is in the navigation menu" src="/assets/img/pro-user-guide/navigating-as-pro-staff.png" class="pro-screenshot" />
+
+If your Administrator has already given you staff permissions, you’ll see additional admin button
+on the top menu when you’re logged in. If not, your Administrator can add these permissions at
+any time: see ‘[Creating a Staff Account](#adminstrating-staff-accounts)’.
+</div>
+
+<div class="pro-task" markdown="1" id="viewing-reports-your-area">
+
+### Viewing all reports in your area
+
+<span class="pro-task__permissions">Permissions required: none</span>
+
+If you want to view everything that’s been reported within the council area, here’s the quickest
+way.
+
+Click on ‘all reports’ in the top menu. You’ll see the boundaries of your council area on the map,
+and a list of all reports listed, with the most recent reports at the top, on the left hand panel.
+
+<img alt="Click all reports in the top menu to view all reports in an area" src="/assets/img/pro-user-guide/viewing-reports-your-area.png" class="pro-screenshot" />
+
+Note that staff members can be allocated one or more category (e.g. potholes, streetlights, etc) by
+the Administrator — see ‘[Creating a Staff Account](#adminstrating-staff-accounts)’ for more details. When logged in, these staff
+members will see reports within these categories by default.
+
+</div>
+
+<div class="pro-task" markdown="1" id="defining-area-interest">
+
+### Defining the area of interest
+
+<span class="pro-task__permissions">Permissions required: none</span>
+
+If required, you can also use the controls to the right of the screen to zoom in or out, or pan across
+the map. As you’ll see, the list of reports always reflects the area you are viewing on the map, so
+this is another way of refining your view.
```

Comment:
Mention using permalink to bookmark once they have something they're happy with?

Reply:
>Once you've found a satisfactory view, and if you are likely to want to revisit it, click on the 'Permalink' button at the bottom of the screen. This will alter the URL displayed in your browser, and allow you it as a bookmark which you may revisit, so you don't have to define the location again in the future.

Reply:
Updated